smartling-attachments-api/src/main/java/com/smartling/api/attachments/v2/AttachmentsApi.java

```diff
@@ -0,0 +1,51 @@
+package com.smartling.api.attachments.v2;
+
+import com.smartling.api.attachments.v2.pto.AttachmentEntitiesPTO;
+import com.smartling.api.attachments.v2.pto.AttachmentListFilter;
+import com.smartling.api.attachments.v2.pto.AttachmentPTO;
+import com.smartling.api.attachments.v2.pto.EntityAttachmentsPTO;
+import com.smartling.api.attachments.v2.pto.EntityLinkCommand;
+import com.smartling.api.attachments.v2.pto.EntityListCommand;
+import com.smartling.api.v2.response.ListResponse;
+import com.smartling.api.v2.response.EmptyData;
+
+import javax.ws.rs.BeanParam;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+@Path("/attachments-api/v2")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public interface AttachmentsApi
+{
+    @GET
+    @Path("/accounts/{accountUid}/{type}/attachments")
+    ListResponse<AttachmentEntitiesPTO> listAttachments(@PathParam("accountUid") String accountUid, @PathParam("type") String type, @BeanParam AttachmentListFilter listFilter);
+
+    @PUT
+    @Path("/accounts/{accountUid}/{type}")
+    ListResponse<EntityAttachmentsPTO> linkAttachments(@PathParam("accountUid") String accountUid, @PathParam("type") String type, EntityLinkCommand command);
```

Comment:
Is this one admin only rigtht now?  I feel like we can exclude this one from the public docs?